### ISSUE TITLE
What's Wrong with my TensorFlow (1.14 or 1.15) implementation?

### ISSUE THREAD

**Issue by guotong1988** [dropreg/R-Drop#13]
![image](https://user-images.githubusercontent.com/4702353/129358131-35e00fad-dc1c-485a-9188-29e40d91b0e3.png)
![image](https://user-images.githubusercontent.com/4702353/129358205-f7dc908d-5003-45d1-af26-259840296979.png)

**Comment by apeterswu** [comment]
Hi, 

Our R-Drop regularization is adopted on the output probability, it seems you are working on the logits (before softmax)?
You could also refer to this third-party implemented tensorflow R-Drop. 
https://github.com/bojone/r-drop

Hope it can be useful.

**Comment by guotong1988** [comment]
Thank you!
